docs/source/topics/ref_dns-configuration.adoc

```diff
@@ -4,11 +4,11 @@
 [id="dns-configuration-general{context}"]
 == General DNS setup
 
-The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
-The `crc.testing` domain is for core OpenShift services.
-The `apps-crc.testing` domain is for accessing OpenShift applications deployed on the cluster.
+The {ocp} cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+The `crc.testing` domain is for core {ocp} services.
+The `apps-crc.testing` domain is for accessing {openshift} applications deployed on the cluster.
 
-For example, the OpenShift API server is exposed as `api.crc.testing` while the OpenShift console is accessed as `console-openshift-console.apps-crc.testing`.
+For example, the {ocp} API server is exposed as `api.crc.testing` while the {ocp} console is accessed as `console-openshift-console.apps-crc.testing`.
```

Comment:
@code-ready/crc-team  Is it accurate to refer to the OpenShift Container Platform API server, or should we leave this as OpenShift API server?

Reply:
I have a hard time differentiating between the 2 names. OCP API server seems fine, but I would not mind the other one eithre ;)

Reply:
In this case it is not a necessity.